en/dronecan/README.md

```diff
@@ -0,0 +1,139 @@
+# DroneCAN
+
+[DroneCAN](https://dronecan.github.io/) is a open software communication protocol for flight controllers and other [CAN](../can/README.md) devices on a vehicle to communicate with each other.
+
+:::note
+PX4 requires an SD card to enable dynamic node allocation and for firmware update.
+The SD card is not used in flight.
+:::
+
+:::note
+DroneCAN was previously known as UAVCAN v0 (or just UAVCAN).
+The name was changed in 2022.
+:::
+
+## Benefits of DroneCAN
```

Comment:
@coderkalyan FYI only, I pulled the "benefits" out of the heading above and made them a section with its own list. I wasn't entirely sure of this, but it does make things easier to read, and means that if we want to say in other places "CAN has many benefits" we have a good place to link.

Reply:
Sure, sounds good.